### PR TITLE
Add receive plain text feature

### PR DESCRIPTION
**Pull request by PauloRSF**
As specified on #88.

I did it using javascript with ajax on the client side to keep the server code uniform.

### CODE REVIEW

File: pages/pages.go [hunk 1]
```diff
@@ -181,19 +181,62 @@ var Upload = `
...
+
+            var formData = new FormData(uploadForm)
+            var titleInput = document.getElementById('plaintext-title')
+            var textInput = document.getElementById('plaintext-text')
+
+            if (titleInput.value || textInput.value) {
+                var currentDate = new Date().toJSON().slice(0,19).replace(/[-T]/g,'_')
+                // If the user didn't specify a file name, use 'qrcp-text-file-${currentDate}'
+                var filename = titleInput.value || ("qrcp-text-file-" + currentDate)
+                var blob = new Blob([textInput.value], { type: "text/plain" })
```
Comment (claudiodangelis) [inline pages/pages.go:231]: I think a newline should be added ad the end of file.

File: pages/pages.go [hunk 2]
```diff
@@ -181,19 +181,62 @@ var Upload = `
             </svg>
         </div>
         <div class="row">
-            <form action="{{.Route}}" method="post" enctype="multipart/form-data">
+            <form id="upload-form">
+                <div class="form-group">
```
Comment (claudiodangelis) [inline pages/pages.go:185]: I would move the "File upload" section above the "Text upload", and would make it clearer that those are two different modes.

```
Send fiels (title)
  Files to transfer (label + button)
Send text (title)
  Title, Text (labels + inputs)
Transfer (button)
```

Ideally, the _Send text_ section should be collapsible and hidden by default so we don't waste space.
The page uses Bootstrap so it shouldn't be hard to implement.

File: pages/pages.go (hunk 1 above)
Comment (PauloRSF) [reply pages/pages.go:231]: Yeah, i agree.

File: pages/pages.go (hunk 2 above)
Comment (PauloRSF) [reply pages/pages.go:185]: Ok, sounds good!

**Comment by claudiodangelis** [comment]
I've done a quick change to the phrasing, think it's ready to be merged now. 
Thank you again for your effort on this! I'm sure users will find it useful, cheers!

**Pull request merged**